app/global.scss

```diff
@@ -0,0 +1,132 @@
+:root {
+  --main-color: #242424;
+  --secondary-color: #ff3393;
+  --font-size-big: 40px;
+  --font-size-medium: 20px;
+}
+
+html {
+  box-sizing: border-box;
+  font-size: 16px;
+}
+
+body,
+h1,
+h2,
+h3,
+h4,
+h5,
+h6,
+p,
+ol,
+ul {
+  margin: 0;
+  padding: 0;
+  font-weight: normal;
+}
+
+ol,
+ul {
+  list-style: none;
+}
+
+img {
+  max-width: 100%;
+  height: auto;
+}
+
+*,
+*:before,
+*:after {
+  box-sizing: inherit;
+}
+
+@font-face {
+  src: url('../public/fonts/catamaran/catamaran bold.ttf');
+  font-family: 'UpLeveled heading font';
+  font-style: bold;
+  font-weight: 800;
+}
+
+@font-face {
+  src: url('../public/fonts/bitstream/VeraMono.ttf');
+  font-family: 'UpLeveled body';
+  font-style: normal;
+}
+
+@font-face {
+  src: url('../public/fonts/bitstream/VeraMono-Bold.ttf');
+  font-family: 'UpLeveled span';
+  font-style: bold;
+}
```

Comment:
should [`@next/font/local`](https://beta.nextjs.org/docs/optimizing/fonts#local-fonts) be used here instead?